server/prometheus.js

```diff
@@ -7,6 +7,15 @@ const commonLabels = [
     "monitor_url",
     "monitor_hostname",
     "monitor_port",
+    "location",
+    "region",
+    "datacenter",
+    "cloud_provider",
+    "az",
+    "rack",
+    "shelf",
+    "room",
+    "floor"
```

Comment:
`"floor",`

Reply:
ah, is this another JS rule I need to be aware of? I'd usually leave off the trailing comma.

Thanks for the comments, I'll address them shortly! :)